packages/localstorage-adapter/src/adapter/adapter.ts

```diff
@@ -70,10 +70,10 @@ const normalizeFlag = (
   // Multi variate flags contain a string or `null` - `false` seems more natural.
   flagValue === null || flagValue === undefined ? false : flagValue,
 ];
-const normalizeFlags = (rawFlags: Readonly<TFlags>) => {
-  if (!rawFlags) return {};
```

Comment:
Was this line removed on purpose? 🧐

Reply:
Thanks for checking in. Yes, with an explicit return it is otherwise a type error as it is not a record. The other normalizeFn (each adapter has one) do not have it. I can't recall why I added this.

Reply:
Actually a test catches me. So I need to refresh my memory a bit ;)